Country.py

```diff
@@ -0,0 +1,270 @@
+            if self.infected_pop >= self.max_pop:
+                self.infected_pop = self.max_pop
+
+        """return integer part of infected population"""
+        # return int(self.infected_pop)
+
+        """return probability"""
+        return self.infected_ratio()
+
+
+    def draw(self):
+        """ draws the location of the country as a circle """
+        pygame.draw.circle(screen, self.color, (int(self.x), int(self.y)),
+                           self.radius)
+
+    def contains_pt(self, pt):
+        """ countains points function to see if mouseclicks are
+        within the circles of the country """
+        x, y = pt
+        if not self.x - self.radius < x < self.x + self.radius:
+            return False
+        if not self.y - self.radius < y < self.y + self.radius:
+            return False
+        return True
+
+    def propagation(self, other):
+        """
+        The pathogen can propagate to other countries with certain probability.
+        """
+        if self.infected_pop >= 1 and other.infected_pop == 0:
+            if random.random() <= self.infected_ratio()/10:
+                other.infected_pop = 1
+
+
+background_color = (255,255,255)
+width, height = 640, 480
+
+screen = pygame.display.set_mode((width, height))
+pygame.display.set_caption('Plague Simulation')
+screen.fill(background_color)
+
+intro = True
+
+while intro:
+    """ Intro of the game gives the intro screen with
+    upgrade instructions as well as how to start the game """
+
+    for event in pygame.event.get():
+        if event.type == pygame.QUIT:
+            pygame.quit()
+            quit()
+        if event.type == pygame.KEYDOWN:
+            if event.key == pygame.K_c:
+                intro = False
+                running = True
+            if event.key == pygame.K_q:
+                pygame.quit()
+                quit()
+    """ Renders the text where it tells the user to
+    press c to start the game, and gives instructions on how to upgrade the game"""
+
+    basicfont = pygame.font.SysFont(None, 20)
+    text = basicfont.render('Welcome To A Plague Simulation! Press C to Start, Click on a country to start your infection', True, (0, 0, 0), (255, 255, 255))
+    textrect = text.get_rect()
+    textrect.centerx = screen.get_rect().centerx
+    textrect.centery = screen.get_rect().centery
+    screen.blit(text, textrect)
+    screen.blit(basicfont.render('For Upgrades, Press I to increase infection rate, K to increase kill rate, and A to increase airborne rate' , True, (0, 0, 0), (255, 255, 255)), (0, 300 ))
+
+    pygame.display.update()
+
+screen = pygame.display.set_mode((640, 480))
+
+""" Herein, three countries are defined """
+country1 = Country(200, 120, 200, radius=60)
+country2 = Country(500, 180, 200, radius=40, color=BLUE)
+country3 = Country(320, 360, 200, radius=50, color=GREEN)
+countries = [country1, country2, country3]
+country_pop_index = country1
+total_pop = 0
+
+Time = 0
+time = 0
+Upgrade_Point = 0
```

Comment:
[By convention](https://www.python.org/dev/peps/pep-0008/#global-variable-names), variables whose values change (variables that are *not constants*) are all lowercase.